yax/modules/identify_informative_hits.py

```diff
@@ -0,0 +1,217 @@
+from yax.shared.utilities import taxidtool
+from yax.artifacts import TaxIDData, InformativeAlignmentData, IdentifiedTaxa
+from yax.state.type.parameters import Int, File
+
+
+def main(working_dir, output, details, identified_taxa_art: IdentifiedTaxa,
+         tax_id_data_art: TaxIDData, lca_dist: Int, min_hits: Int,
+         min_relative_hits: Int, sequences_input_fp: File) \
+            -> InformativeAlignmentData:
```

Comment:
sort of yes, if you are asking what i think you are asking: that is done in the write tree component of the artifact completion... though that is really ancillary and more of a debug for our purposes